rust/src/traits/samplers/uniform/mod.rs

```diff
@@ -178,9 +178,9 @@ impl SampleUniformIntBelow for UBig {
         loop {
             fill_bytes(&mut buffer)?;
 
-            let sample = UBig::from_be_bytes(&buffer);
-            if sample < threshold {
-                return Ok(sample % &upper);
+            let v = UBig::from_be_bytes(&buffer);
+            if v < threshold {
+                return Ok(v % &upper);
```

Comment:
The longer name didn't seem like a bad thing to me... and `v` is just unusual enough that I wonder if there are some semantics you are trying to communicate.